spring-reactive-modules/spring-5-reactive-client-2/src/main/java/com/baeldung/limitrequests/client/utils/Client.java

```diff
@@ -0,0 +1,15 @@
+package com.baeldung.limitrequests.client.utils;
+
+public interface Client {
+    String separator = ":";
```

Comment:
Since this an attribute of an interface is like a constant, so the name should be in uppercase.